scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -1044,7 +1074,7 @@ static int boost_lat(struct task_struct *p, struct task_ctx *taskc,
 		     struct cpu_ctx *cpuc, bool is_wakeup)
 {
 	u64 run_time_ft = 0, wait_freq_ft = 0, wake_freq_ft = 0;
-	u64 lat_cri_raw = 0;
+	u64 lat_cri_raw, perf_cri_raw;
```

Comment:
Doesn't look like `perf_cri_raw` is used anywhere?

Reply:
Ooops. Fixed.